text-serializer-gson-legacy-impl/build.gradle

```diff
@@ -0,0 +1,11 @@
+dependencies {
+  api(project(":adventure-api"))
+  api(project(":adventure-text-serializer-gson"))
+  api(project(":adventure-nbt"))
+}
+
+jar {
+  manifest.attributes(
+    "Automatic-Module-Name": "net.kyori.adventure.text.serializer.gson.legacyhovereventimpl"
```

Comment:
This doesn't really match the artifact id or the name of the package -- should probably be aligned more?